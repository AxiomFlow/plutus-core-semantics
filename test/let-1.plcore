(let 2 x (builtin addInt (x x)))
