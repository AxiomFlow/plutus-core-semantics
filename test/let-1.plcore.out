<T> <k> 4 </k> </T>
